eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubSnpNoHgvsToClusteredVariantProcessorTest.java

```diff
@@ -0,0 +1,360 @@
+/*
+ * Copyright 2014-2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp.processors;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import uk.ac.ebi.eva.accession.dbsnp.model.DbsnpVariantType;
+import uk.ac.ebi.eva.accession.dbsnp.model.Orientation;
+import uk.ac.ebi.eva.accession.dbsnp.model.SubSnpNoHgvs;
+import uk.ac.ebi.eva.accession.dbsnp.persistence.DbsnpClusteredVariantEntity;
+import uk.ac.ebi.eva.commons.core.models.VariantType;
+
+import java.sql.Timestamp;
+
+import static org.junit.Assert.assertEquals;
+
+public class SubSnpNoHgvsToClusteredVariantProcessorTest {
+
+    private static final String ASSEMBLY = "AnyAssembly-1.0";
+
+    private static final String PROJECT_ACCESSION = "HANDLE_NAME";
+
+    private static final String BATCH_HANDLE = "HANDLE";
+
+    private static final String BATCH_NAME = "NAME";
+
+    private static final String CHROMOSOME = "10";
+
+    private static final long CHROMOSOME_START = 1000000L;
+
+    private static final String CONTIG_NAME = "Contig1";
+
+    private static final long CONTIG_START = 20000L;
+
+    private static final int TAXONOMY = 9999;
+
+    private static final Timestamp CREATED_DATE = Timestamp.valueOf("2001-01-05 12:30:50.0");
+
+    SubSnpNoHgvsToClusteredVariantProcessor processor;
+
+    @Before
+    public void setUp() throws Exception {
+        processor = new SubSnpNoHgvsToClusteredVariantProcessor();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+    }
+
+    @Test
+    public void transformSnpForwardOrientations() throws Exception {
+        SubSnpNoHgvs subSnpNoHgvs = new SubSnpNoHgvs(25928972L, 14718243L, "A", "A/C", ASSEMBLY, BATCH_HANDLE,
+                                                     BATCH_NAME, CHROMOSOME, CHROMOSOME_START, CONTIG_NAME,
+                                                     CONTIG_START, DbsnpVariantType.SNV, Orientation.FORWARD,
+                                                     Orientation.FORWARD, Orientation.FORWARD, false, false, false,
+                                                     false, CREATED_DATE, CREATED_DATE, TAXONOMY);
+
+        DbsnpClusteredVariantEntity variant = processor.process(subSnpNoHgvs);
+        // TODO: validated, match assembly and RS variant accession are being added into PR #28
```

Comment:
I think that some of this TODO comments can be resolved now (at least the RS variant accession and the validated ones). Add the assertions if needed to the assertProcessedVariant method.